src/test/kotlin/uk/gov/justice/digital/hmpps/calculatereleasedatesapi/resource/CalculationControllerTest.kt

```diff
@@ -259,17 +195,4 @@ class CalculationControllerTest {
 
     assertThat(result.response.contentAsString).isEqualTo(mapper.writeValueAsString(breakdown))
   }
-
```

Comment:
This test was incorrect.. calculationTransactionalService.findUserInput can not return null